source/scenes/gameplay/PPColorStrip.cpp

```diff
@@ -9,7 +9,7 @@ ptr<ColorStrip> ColorStrip::alloc(
     return (result->init() ? result : nullptr);
 }
 
-void ColorStrip::update(const vec<uint> &canvasColors) {
+void ColorStrip::update(const vec<uint> &canvasColors, bool colorblind) {
```

Comment:
Note I had to touch a lot of files to add this "bool colorblind" parameter. I did this to avoid a PPColorStrip -> SaveController dependency, but if that's okay I can just switch it to that

Reply:
I think it might be better to allow that dependency - being a singleton class, i don't think there's any harm in directly accessing the SaveController instance from different files.

Reply:
Done!